doc/lib/nf3.mjs.html

```diff
@@ -727,7 +727,7 @@ <h1 class="page-title">Source: nf3.mjs</h1>
   value of each propery is the number of tokens originating from that contract.
   */
   async getLayer2Balances() {
-    const res = await axios.get(`${this.clientBaseUrl}/commitment/balance`);
+    const res = await axios.post(`${this.clientBaseUrl}/commitment/balance`);
```

Comment:
Updating to `GET`.

Probably a rebase with the ERC1155 branch will update this automatically.